diff -ruN orig/resources/linux/atom.desktop.in new/resources/linux/atom.desktop.in
--- orig/resources/linux/atom.desktop.in	2019-06-12 21:42:52.000000000 -0600
+++ new/resources/linux/atom.desktop.in	2019-06-16 21:32:01.378080828 -0600
@@ -2,8 +2,8 @@
 Name=<%= appName %>
 Comment=<%= description %>
 GenericName=Text Editor
-Exec=env ATOM_DISABLE_SHELLING_OUT_FOR_ENVIRONMENT=false <%= installDir %>/bin/<%= appFileName %> %F
-Icon=<%= iconPath %>
+Exec=env ATOM_DISABLE_SHELLING_OUT_FOR_ENVIRONMENT=false /usr/share/atom/atom %F
+Icon=atom
 Type=Application
 StartupNotify=true
 Categories=GNOME;GTK;Utility;TextEditor;Development;
