--- script/lib/create-rpm-package.js	2017-06-14 11:29:29.000000000 -0600
+++ create-rpm-package-new.js	2017-07-27 10:48:53.272779730 -0600
@@ -19,7 +19,7 @@
   // (Ref.: https://twiki.cern.ch/twiki/bin/view/Main/RPMAndDebVersioning)
   const appVersion = CONFIG.appMetadata.version.replace(/-/g, '.')
 
-  const rpmPackageDirPath = path.join(CONFIG.homeDirPath, 'rpmbuild')
+  const rpmPackageDirPath = path.join(CONFIG.homeDirPath, 'build/BUILD/rpm_out')
   const rpmPackageBuildDirPath = path.join(rpmPackageDirPath, 'BUILD')
   const rpmPackageSourcesDirPath = path.join(rpmPackageDirPath, 'SOURCES')
   const rpmPackageSpecsDirPath = path.join(rpmPackageDirPath, 'SPECS')
@@ -69,16 +69,4 @@
     path.join(CONFIG.repositoryRootPath, 'atom.sh'),
     path.join(rpmPackageBuildDirPath, 'atom.sh')
   )
-
-  console.log(`Generating .rpm package from "${rpmPackageDirPath}"`)
-  spawnSync('rpmbuild', ['-ba', '--clean', rpmPackageSpecFilePath])
-  for (let generatedArch of fs.readdirSync(rpmPackageRpmsDirPath)) {
-    const generatedArchDirPath = path.join(rpmPackageRpmsDirPath, generatedArch)
-    const generatedPackageFileNames = fs.readdirSync(generatedArchDirPath)
-    assert(generatedPackageFileNames.length === 1, 'Generated more than one rpm package')
-    const generatedPackageFilePath = path.join(generatedArchDirPath, generatedPackageFileNames[0])
-    const outputRpmPackageFilePath = path.join(CONFIG.buildOutputPath, `atom.${generatedArch}.rpm`)
-    console.log(`Copying "${generatedPackageFilePath}" into "${outputRpmPackageFilePath}"`)
-    fs.copySync(generatedPackageFilePath, outputRpmPackageFilePath)
-  }
 }
